src/data/members.json

```
@@ -23,5 +23,10 @@
 		"displayName": "Kai",
 		"link": null,
 		"image": null
-	}
-}
+	},
+	"shibaki": {
+		"displayName": "Shibaki",
+		"link": null,
+		"image": null
```

[imo]

もし画像を追加するのであれば `public/members/` 以下に画像ファイルを追加すると↓みたいに書けばアクセスできます！

```suggestion
		"image": "https://uyupun.github.io/takada-semi/members/shibaki.拡張子"
```